extensions/PhabBugz/lib/Util.pm

```diff
@@ -536,4 +537,41 @@ sub add_security_sync_comments {
     Bugzilla->set_user($old_user);
 }
 
+sub get_needs_review {
+    my ($user) = @_;
+    $user //= Bugzilla->user;
+    return unless $user->id;
+
+    my $ids = get_members_by_bmo_id([$user]);
+    return [] unless @{$ids};
+    my $phid_user = $ids->[0];
+
+    my $diffs = request(
+        'differential.revision.search',
+        {
+            attachments => {
+                reviewers => 1,
+            },
+            constraints => {
+                reviewerPHIDs => [$phid_user],
+                statuses      => [qw( needs-review )],
+            },
+            order       => 'newest',
```

Comment:
Phab's api's default limit is 100 on searches. I'm not sure how fast/slow the dashboard is so we could limit that further here if needed. Hopefully, no one lets their review queue get that big!

Reply:
100 rows isn't a problem for the dashboard performance-wise, but it's a reasonable cutoff for this table.